filetablemodel.cpp

```diff
@@ -88,6 +89,14 @@ bool FileTableModel::readFile(const QString &fileName)
 
     return true;
 }
+void FileTableModel::fillHashTypes(const QStringList &listHashTypes)
+{
+    for(int i=0; i < listHashTypes.size() && i< rowCount() ; i++)
+    {
+        // Possible hash types is column at 4
+        setData(index(i,4),listHashTypes[i]);
```

Comment:
You are not following the code style here. Maybe it would be worth reading the links provided in the roadmap again from top to bottom.